core/build.gradle.kts

```diff
@@ -59,44 +45,23 @@ kotlin {
 publishing {
     publications {
         create<MavenPublication>("coreJVM") {
-            groupId = "codes.laurence.warden"
-            artifactId = "warden-core"
-            version = version
+            artifact(tasks.jvmJar)
+            artifact(tasks.jvmSourcesJar)
 
-            artifact("$buildDir/libs/warden-core-jvm-${project.version}-sources.jar") {
-                classifier = "sources"
-            }
-            artifact("$buildDir/libs/warden-core-jvm-${project.version}.jar")
+            createWardenPom()
         }
     }
 }
 
 artifactory {
-    setContextUrl("https://laurencecodes.jfrog.io/artifactory")
-    publish(
-        delegateClosureOf<org.jfrog.gradle.plugin.artifactory.dsl.PublisherConfig> {
-            repository(
-                delegateClosureOf<org.jfrog.gradle.plugin.artifactory.dsl.DoubleDelegateWrapper> {
-                    setProperty("repoKey", "codes.laurence.warden")
-                    setProperty("username", System.getenv("JFROG_USER"))
-                    setProperty("password", System.getenv("JFROG_PASSWORD"))
-                    setProperty("maven", true)
-                }
-            )
-            defaults(
-                delegateClosureOf<org.jfrog.gradle.plugin.artifactory.task.ArtifactoryTask> {
-                    publications("coreJVM")
-                }
-            )
+    publish {
+        defaults {
+            publications("coreJVM")
         }
-    )
+    }
 }
 
-tasks {
-    val build by existing
-
-    artifactoryPublish {
-        dependsOn(build)
-        dependsOn("publishJvmPublicationToMavenLocal")
-    }
+tasks.artifactoryPublish.configure {
+    dependsOn(tasks.build)
+//    dependsOn(tasks.publishJvmPublicationToMavenLocal)
```

Comment:
I think a dependency on `tasks.publishJvmPublicationToMavenLocal` is unnecessary? But maybe there's a reason for it.

Reply:
If I remember correctly it was grabbing the output jar from the build directory to push to artifactory, but if we are moving to maven we can scrap the artifactory stuff. Will see how we go when we swap in the sonatype repo in.